packages/js/src/installation-success.js

```diff
@@ -117,7 +118,12 @@ function InstallationSuccessPage() {
 							</a>
 						</div>
 					</div>
-					<a id="installation-success-skip-link" className="yst-bottom-12 yst-right-0 yst-mr-5 yst-self-end yst-text-base md:yst-absolute" href={ "/wp-admin/admin.php?page=wpseo_dashboard" }>
+					<a
+						id="installation-success-skip-link"
+						className="yst-bottom-12 yst-right-0 yst-mr-5 yst-self-end yst-text-base md:yst-absolute"
+						href={ "/wp-admin/admin.php?page=wpseo_dashboard" }
+						data-hiive-event-name="clicked_skip_button | installation successful screen"
+					>
 						{
 							/* translators: %s expands to ' »'. */
 							sprintf( __( "Skip%s", "wordpress-seo" ), " »" )
```

Comment:
Not part of your PR. However, I think we should not put whitespace in an untranslatable string due to RTL languages. In fact, this whole `sprintf` is a bit strange to me. Why is this not just up to translators? Ah well, I digress.

Reply:
Investigated a little:
* The original PR does not note any reason why we do it like this: https://github.com/Yoast/wordpress-seo/pull/17789
* When you go to `/wp-admin/admin.php?page=wpseo_installation_successful_free` and check the Skip in the bottom right. And then switch to RTL, it looks totally fine in the bottom left!
* Must be the browser doing things? 🪄 
So let's leave it 😉